src/calculateRentalCost.js

```diff
@@ -4,7 +4,19 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const TOTAL_AMOUNT = days * 40;
```

Comment:
it's not a constant

Reply:
I've tried to use "let" and this error in VS
<img width="618" alt="2" src="https://github.com/user-attachments/assets/51e6b9df-158b-4c7d-9ba0-3c2adf65bd22">

Reply:
Just watched video with Solution of this task and i get what you mean. Thank you!